core/src/main/resources/lib/hudson/buildHealth.jelly

```diff
@@ -65,13 +65,13 @@ THE SOFTWARE.
              </j:choose>  
         </j:if>
         <j:if test="${!empty(healthReports)}">
-            <div class="healthReportDetails">
-                <table border="0">
+            <div class="jenkins-tooltip healthReportDetails">
+                <table class="bigtable" border="0">
```

Comment:
That makes the temporary and (in my opinion) ugly `bigtable` style even more prominent. But that is a matter of taste and only my personal opinion.

Reply:
I've only done it to make the two tooltips (on the home page and the project page) consistent. Is there any intention to kill `.bigtable`? I wouldn't mind taking a look at replacing it myself in the future as like you said, it isn't the nicest thing in the world.

Reply:
I understand. Then make it consistent! We can improve the `.bigtable` in another step in the future. It would be awesome if you are willing to help!   

Last year Cloudbees UX team started to re-designed tables (and tabs) but then the team disappeared before finishing (or polishing) the result. Now we have some intermediate UI design which does look better than before but still is far from perfect.